openc3/python/openc3/io/stderr.py

```diff
@@ -28,6 +29,6 @@ def __init__(self):
     # @return [Stdout] Returns a single instance of Stdout
     @classmethod
     def instance(cls):
-        if not cls.my_instance:
+        if not Stderr.my_instance:
```

Comment:
This is clearer so I like the change. Is it equivalent to the old `cls.my_instance`?